apps/ff-site/redirects.mjs

```diff
@@ -193,7 +193,12 @@ export const redirects = [
   },
   {
     source: '/ecosystem-explorer/solmedia/%E2%80%A6',
-    destination: '/ecosystem-explorer/solmedia',
+    destination: '/ecosystem-explorer',
+    permanent: true,
+  },
```

Comment:
What is this for?

Reply:
Ah, I see now why it was confusing. Would something like this work? To refactor the two?

```
{
  source: '/ecosystem-explorer/solmedia/:path*',
  destination: '/ecosystem-explorer',
  permanent: true,
}
```